.github/workflows/php-tests.yml

```diff
@@ -43,3 +49,11 @@ jobs:
 
       - name: Run sample
         run: php sample_push.php | grep "Trying tls://gateway.sandbox.push.apple.com:2195"
+
+      - name: Setup PCOV
+        run: |
+          composer require pcov/clobber
+          vendor/bin/pcov clobber
+
```

Comment:
> If you are using PHPUnit 5.x, 6.x or 7.x, you need to set up pcov/clobber before executing your tests.

We're using 9.x so:
```suggestion
```

Reply:
Yea I see that now. I thought it said php 8. Reading is hard...